src/services/Models.service.tsx

```diff
@@ -287,12 +286,17 @@ function buildFallback(nodes: Nodes, transform: Transform = new Transform()): JS
         if (node.type === 'Mesh') {
           return (
             <mesh
+              attach={(parent, self) => {
+                parent.add(self);
+                self.parent = node.parent;
+                return () => parent.remove(self);
+              }}
```

Comment:
Add the mesh as a child to the <group>, but reparent the nodes object3D to the object3D of its parent node